pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -376,40 +390,57 @@ def default_compilation_pass(
         :return: Compilation pass guaranteeing required predicates.
         :rtype: BasePass
         """
+        config: QasmBackendConfiguration = self._backend.configuration()
+        props: Optional[BackendProperties] = self._backend.properties()
+        return IBMQBackend.default_compilation_pass_static(
+            config, props, optimisation_level, placement_options
+        )
+
+    @staticmethod
+    def default_compilation_pass_static(
```

Comment:
maybe rename this as well to `...offline` ?

Reply:
Done. Also changed `rebase_pass_static` to `rebase_pass_offline` for consistency.